app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/DevelopmentPreviewWrapper.kt

```diff
@@ -64,7 +63,7 @@ fun DevelopmentPreviewWrapper(
                     .background(RadixTheme.colors.orange2)
                     .statusBarsPadding()
                     .onGloballyPositioned { coordinates ->
-                        bannerHeight = with(density) { coordinates.size.height.toDp() }
+                        devBannerHeight.height.value = with(density) { coordinates.size.height.toDp() }
```

Comment:
It seem that documentation discourages [this pattern](https://developer.android.com/develop/ui/compose/phases#recomp-loop) (see bottom part of that section), but I am not sure how we could re-implement that, as previous implementation also uses same pattern

Reply:
Yes the only possible solution is to create some custom layout for this, since Column does not work for us in this case.